src/test/scala/NodeTest.scala

```diff
@@ -0,0 +1,37 @@
+import link_list.com.Node
+import org.scalatest.funsuite.AnyFunSuite
+
+class NodeTest extends AnyFunSuite{
+ test("Node.isCirle When linked_list is not circle"){
+   val head= new Node()
+   val  t1=  head.setNext(Some(head))
+   val t2=  t1.setNext(Some(new Node()))
```

Comment:
node của em lạ vậy, muốn append 1 node vào cuối phải chạy hết toàn bô node à. linked node thế này chưa đúng?